homeassistant/components/vallox/__init__.py

```diff
@@ -125,7 +125,7 @@ def get_metric(self, metric_key: str) -> StateType:
     @property
     def model(self) -> str | None:
         """Return the model, if any."""
-        model = cast(str, _api_get_model(self.metric_cache))
+        model = _api_get_model(self.metric_cache)
```

Comment:
This is an odd change combined with the upstream change.

Upstream now returns `Unknown` as the model, however, if the model is unknown, this property method should return `None` in Home Assistant.

Reply:
Hmm, isn't that what the `if` below it does? 🤔 Or am I misunderstanding you?

Reply:
LOL no, you understood right. I just didn't look well enough 🤦 . Sorry!

Reply:
No worries 😀 👍